wcivf/apps/elections/urls.py

```diff
@@ -74,4 +74,12 @@
         ),
         name="sv_voting_system_view",
     ),
+    re_path(
+        r"^voting_system/stv/",
+        TranslatedTemplateView.as_view(
```

Comment:
Do you need to rerun `makemessages`? Or is it not necessary for `TranslatedTemplateView`?

Reply:
No, for this we just email the HTML file to the translators and paste their response in a `filename_[language].html` file